pkg/minikube/config/config_test.go

```diff
@@ -141,3 +142,43 @@ func Test_readConfig(t *testing.T) {
 		t.Errorf("Did not read config correctly,\n\n wanted %+v, \n\n got %+v", expectedConfig, mkConfig)
 	}
 }
+
+func Test_writeConfig(t *testing.T) {
+	configFile := "./testdata/configTest"
+	cfg := map[string]interface{}{
+		"vm-driver":            constants.DriverKvm2,
+		"cpus":                 4,
+		"disk-size":            "20g",
+		"show-libmachine-logs": true,
+		"log_dir":              "/etc/hosts",
+	}
+
+	err := writeConfig(configFile, cfg)
```

Comment:
Can this test instead write to an out-of-tree path by using `ioutil.TempFile`? This appears to update the git tree.